packages/docs/content/docs/tutorials/supabase-mcp-server.mdx

```diff
@@ -0,0 +1,584 @@
+
+<div
+  style={{
+    position: "relative",
+    paddingBottom: "53.7%",
+    marginBottom: 20,
+    height: 0,
+    overflow: "hidden",
+    maxWidth: "100%",
+  }}
+>
+  <iframe
+    src="https://drive.google.com/file/d/1tqM2tglxVkw3gNUR0TYJiPWgOmfezS1d/preview"
+    style={{
+      position: "absolute",
+      top: 0,
+      left: 0,
+      width: "100%",
+      height: "100%",
+      border: "none",
+      borderRadius: "8px",
+    }}
+    allowFullScreen
+    title="Testing MCP Server in MCP Inspector"
+  />
+</div>
+
+
+### Step 6: Executing the MCP server in Pochi
+
+Once it works in Inspector, we can connect it into Pochi (your config already points to `index.js`). Then when you give Pochi a natural language prompt,  asking the model to call `queryDatabase` tool, it will answer all your database related queries easily. 
+
+Prompt used: 
+
+>Use the `queryDatabase` tool to find out what are the total sales for each product by region
+
+Here’s the final working demo:
+
+<div
+  style={{
+    position: "relative",
+    paddingBottom: "53.7%",
+    marginBottom: 20,
+    height: 0,
+    overflow: "hidden",
+    maxWidth: "100%",
+  }}
+>
+  <iframe
+    src="https://drive.google.com/file/d/1enHFWwNik28f8oWMKtMY5QVCuMd7lbSZ/preview"
+    style={{
+      position: "absolute",
+      top: 0,
+      left: 0,
+      width: "100%",
+      height: "100%",
+      border: "none",
+      borderRadius: "8px",
+    }}
+    allowFullScreen
+    title="Testing MCP Server in MCP Inspector"
+  />
+</div>
+
+You can also paste in the Vega - Lite JSON spec you receive in response into the Vega Editor and get a chart response of the requested query:
+
+![Vega Lite Chart](./images/vega-chart-final-image.png)
+
+## How This Improves Developer Workflow for Teams and Enterprises
+
+While the sample database in this tutorial uses `sales_data` for simplicity, the core idea is universally applicable to developer workflows in enterprise environments.
+
+In real-world projects, your database might contain user accounts and sessions, application logs or event streams, feature flags and configuration tables or order and subscription data.
+Which is why being able to ask questions like:
+
+> Are there any failed jobs in the `background_tasks` table?
+
+> How many users signed up in the last 7 days from Europe?
+
+> What’s the schema of the `sales_data` table?
+
+
+…directly in your code editor helps you debug issues without switching to a DB client, quickly verify backend logic during development and stay in flow while writing or reviewing code.
+
+<div
+  style={{
+    position: "relative",
+    paddingBottom: "53.7%",
+    marginBottom: 20,
+    height: 0,
+    overflow: "hidden",
+    maxWidth: "100%",
+  }}
+>
+  <iframe
+    src="https://drive.google.com/file/d/1oUbk24ZuNbnG3okq1npFqJ2YxxwAUi9w/preview"
+    style={{
+      position: "absolute",
+      top: 0,
+      left: 0,
+      width: "100%",
+      height: "100%",
+      border: "none",
```

Comment:
Consider merging this 'Next Steps' section with the previous 'How This Improves...' section. This would create a single, strong conclusion that summarizes the two main takeaways of the tutorial: 1) harnessing Pochi's AI, and 2) using MCP for powerful customization in team environments. The current next steps can be woven in as brief examples of that customization potential.